src/Facebook/Facebook.php

```diff
@@ -133,6 +133,9 @@ public function __construct(array $config = [])
             'persistent_data_handler' => null,
             'pseudo_random_string_generator' => null,
             'url_detection_handler' => null,
+            'http_request_timeout' => FacebookClient::DEFAULT_REQUEST_TIMEOUT,
+            'http_file_upload_timeout' => FacebookClient::DEFAULT_FILE_UPLOAD_REQUEST_TIMEOUT,
```

Comment:
Are these also defined as 5 seconds? if so they will almost always fail on large file uploads right?

Reply:
They are not: https://github.com/CommentSold/php-facebook-graph-sdk/pull/3/files#diff-fda3992eb4479a5470512553ad5f89ce5baf66762a389de93a4e377067aeae64R71

this will keep the existing behavior but we can change file upload timeouts from the monolith if we choose!